styles/variables-base.less

```diff
@@ -242,9 +242,9 @@
 @agate-heading-margin: 0 @agate-spotlight-outset (1.5 * @agate-spotlight-outset) @agate-spotlight-outset;
 @agate-heading-padding: 0;
 @agate-heading-spacing-title: 0;
-@agate-heading-spacing-medium: @agate-component-spacing;
-@agate-heading-spacing-small: inherit;
-@agate-heading-spacing-large: inherit;
+@agate-heading-spacing-medium: 18px;
+@agate-heading-spacing-small: 12px;
+@agate-heading-spacing-large: 24px;
```

Comment:
Oh, I think it would be better to initialize values based on `@agate-component-spacing`.
And less variables would be ordered alphabetically or meaningfully.
```
@agate-heading-spacing-small: @agate-component-spacing
@agate-heading-spacing-medium: (@agate-component-spacing + 6px)
@agate-heading-spacing-large: (@agate-component-spacing + 12px)
```
or
```
@agate-heading-spacing-large: (@agate-component-spacing + 12px)
@agate-heading-spacing-medium: (@agate-component-spacing + 6px)
@agate-heading-spacing-small: @agate-component-spacing
```